views/random-beer.hbs

```diff
@@ -0,0 +1,5 @@
+<h1>Random Beer</h1>
+{{theBeer}}
```

Comment:
I think this is not needed here. If you want to set the context, you could use `{{#with theBeer}}` and close it with `{{/with}}` but here you don't need to.